### PR TITLE
Add template that looks like https://foss4g-africa.org/en/home

### LINKED ISSUE #35: foss4g-africa.org: skin
1. for now, keep the FOSS4G Bonn 2016 skin and just change wording and graphics for FOSS4G Africa 2017
1. get home page to match the style of the rest of the site. (see current production site)

### PR DESCRIPTION
**Pull request by euclidian**
This fix #35 . 

We have both templates that match https://qgis.org.za and https://foss4g-africa.org
Data can be edited from admin page.

Also changes several wording that still related to 2016 conf

### CODE REVIEW

**Comment by dimasciput** [comment]
Please change this PR to develop branch instead.

**Comment by euclidian** [comment]
commit #353aabe make it possible to resolve home page content based on host

Notes:
- Url metadata for qgis.org.za must match "/" in admin page
- Url metadata for foss4g-africa.org must match "foss4g-home-2017" in admin page

Improvement needed:

- Slug for sites is still hardcoded in core/views.py. We need to move it as metadata in get_current_site() so it can be loaded dynamicalyy

File: django_project/core/views.py [hunk 1]
```diff
@@ -1 +1,75 @@
...
+                                   "settings.MIDDLEWARE")
+
+    if not hasattr(request, "page") or request.page.slug != slug:
+        raise Http404
+
+    # Check for a template name matching the page's slug. If the homepage
+    # is configured as a page instance, the template "pages/index.html" is
+    # used, since the slug "/" won't match a template name.
+    # template_name = str(slug) if slug != home_slug() else "index"
+    if slug == home_slug():
```
Comment (euclidian) [inline django_project/core/views.py]: Please review @dimasciput . Is this best practice for multiple site home page?

Comment (dimasciput) [inline django_project/core/views.py]: I think using get_current_site would be much better.

File: deployment/sites-enabled/default.conf [hunk 2]
```diff
@@ -57,4 +57,10 @@ server {
         uwsgi_param  SERVER_NAME        $server_name;
     }
 
+    location /.well-known/acme-challenge {
```
Comment (dimasciput) [inline deployment/sites-enabled/default.conf]: We don't need this in here anymore.

File: django_project/core/views.py (hunk 1 above)
Comment (euclidian) [inline django_project/core/views.py]: I've been searching way to do this and found a way. Currently an "active site" is a setting in core/settings/base.py in SITE_ID variable. But what we want to achieve is qgis.org.za and foss4g-africa is one instance so we need to read active site based on request. 

To achieve this behaviour we need to create a Midlleware that read a request and set SITE_ID value dynamically based on request. But on the other hand, we need to  query to database for each request to home base just to set SITE_ID. Is this ok?

Comment (dimasciput) [inline django_project/core/views.py]: Please change this based on our discussion.

File: django_project/core/views.py [hunk 3]
```diff
@@ -39,16 +40,15 @@ def page(request, slug, template=u"pages/page.html", extra_context=None):
...
-                                                         include_login_required=True)
-            if pages:
-                home_page = pages[0]
-                setattr(request,"page",home_page)
+        #if host contains staging,https and port, then remove it
+        host = request.get_host().split("//")[-1].split("/")[0].split(":")[0].replace("staging.","")
+        pages = Page.objects.with_ascendants_for_slug(host,
+                                                      for_user=request.user,
+                                                      include_login_required=True)
+        if pages:
```
Comment (euclidian) [inline django_project/core/views.py]: @dimasciput fixed using staging name

**Comment by euclidian** [comment]
Update commit  11cf933

Notes:
- Url metadata for corresponding homepage must equal to its hostname without port, https, htpp, and staging subdomain

**Comment by dimasciput** [comment]
LGTM

**Pull request merged**